src/main/kotlin/eli/inevitable/formatters/CellFormatter.kt

```diff
@@ -0,0 +1,261 @@
+                verticalPaddingCount = formatter.verticalPaddingCount
+                horizontalPaddingCount = formatter.horizontalPaddingCount
+                lineWidth = formatter.lineWidth
+                height = formatter.height
+                width = formatter.width
+            }
+        }
+
+        constructor(): super(CellFormatter())
+
+        constructor(vararg contents: String): this(contents.toList())
+
+        constructor(contents: List<String>): this() {
+            addLines(contents)
+        }
+
+        companion object {
+            @JvmStatic
+            fun of(original: CellFormatter): Builder = Builder(original)
+
+            @JvmStatic
+            fun of(): Builder = Builder()
+
+            @JvmStatic
+            fun of(vararg contents: String): Builder = Builder(contents.toList())
+
+            @JvmStatic
+            fun of(contents: List<String>): Builder = Builder(contents)
+        }
+
+        /**
+         * @param lines The [array of text lines][String] to populate the rows of the cell.
+         * @return This [Builder]
+         */
+        fun withLines(vararg lines: String): Builder {
+            return withLines(lines.toList())
+        }
+
+        /**
+         * @param lines The [list of text lines][String] to populate the rows of the cell.
+         * @return This [Builder]
+         */
+        fun withLines(lines: List<String>): Builder {
+            checkFinalizedStatus()
+            buildable.rawLines.clear()
+
+            return addLines(lines)
+        }
+
+        /**
+         * @param lines The [array of text lines][String] to populate the rows of the cell.
+         * @return This [Builder]
+         */
+        fun addLines(vararg lines: String): Builder {
+            return addLines(lines.toList())
+        }
+
+        /**
+         * @param lines The [list of text lines][String] to populate the rows of the cell.
+         * @return This [Builder]
+         */
+        fun addLines(lines: List<String>): Builder {
+            checkFinalizedStatus()
+            with(buildable) {
+                rawLines.addAll(lines.flatMap { it.sanitize() })
+                width = maxOf(width, rawLines.maxWidth() + 2 * horizontalPaddingCount)
+                height = maxOf(height, rawLines.size + 2 * verticalPaddingCount)
+            }
+
+            return this
+        }
+
+        /**
+         * @param alignment Configures how to align the text within the cell. Default: [Align.LEFT]
+         * @return This [Builder]
+         */
+        fun withAlignment(alignment: Align): Builder {
+            checkFinalizedStatus()
+            buildable.alignment = alignment
+
+            return this
+        }
+
+        /**
+         * @param token The displayable [token][Char] used to fill the spacing between the raw-text and the cell padding. Default: ' '.
+         * @return This [Builder]
+         */
+        fun withFillingSpacer(token: Char): Builder {
```

Comment:
should declaration be defaulted ? doesn't look like its defaulted atm.

Reply:
ah, I see down below where the paddingSpacer stuff gets initialized to the defaults.